apps/builder/app/builder/shared/expression-editor.tsx

```diff
@@ -212,9 +212,14 @@ const autocompletionStyle = css({
 const emptyScope: Scope = {};
 const emptyAliases: Aliases = new Map();
 
+const errorStyle = css({
+  borderColor: theme.colors.borderDestructiveMain,
+});
+
 export const ExpressionEditor = ({
   scope = emptyScope,
   aliases = emptyAliases,
+  error = false,
```

Comment:
why not choosing a similar interface we have in InputField and others:

prop color - defines colors e.g. color=error